asciidoc/courses/drivers-java/modules/2-handling-results/lessons/4c-working-with-dates-and-times/questions/1-datetime-creation.adoc

```diff
@@ -0,0 +1,53 @@
+[.question.select-in-source]
+= Local datetime
+
+Update the code to create a date time object for May 15, 2024 at 2:30 PM in the **UTC+2 timezone**.
+
+[source,Java,role=nocopy noplay]
+----
+var datetime = ZonedDateTime.of(
+    2024, 05, 15, 14, 30, 00, 0, 
+    /*select:ZoneId.of("+02:00")*/
+);
+----
+
+- [ ] "+02:00"
+- [ ] "UTC+2"
+- [ ] ZoneId.of("UTC+2")
+- [x] ZoneId.of("+02:00")
+
+[TIP,role=hint]
+.Hint
+====
+Remember that:
+- The timezone needs to be specified using `ZoneId.of`
+- You need to specify the timezone offset in the format `+hh:mm` or `-hh:mm`
```

Comment:
I believe you can also use the text format of the city/region as the value here: https://neo4j.com/docs/cypher-manual/current/values-and-types/temporal/#cypher-temporal-timezones and https://neo4j.com/docs/java-manual/current/data-types/#_temporal_types. They both follow the Java.time format here: https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/ZoneId.html

Reply:
Thanks, I have updated the answers.